src/test/java/com/snowflake/kafka/connector/internal/IngestionServiceIT.java

```diff
@@ -97,27 +103,53 @@ public void ingestFileWithClientInfoTestSuccessful() throws Exception {
         60,
         9);
     // after ingestion, check if the offset token is updated
-    offsetToken = ingestService.getClientStatus();
-    assert offsetToken.equals("1");
+    clientStatusResponse = ingestService.getClientStatus();
+    assert clientStatusResponse.getOffsetToken().equals("1");
   }
 
   @Ignore
   @Test(expected = SnowflakeKafkaConnectorException.class)
   public void ingestFileWithClientInfoTestFailed() throws Exception {
     String file = "{\"aa\":123}";
+    // File Name Format: app/table/partition/start_end_timeStamp.fileFormat.gz
+    // start offset = 0, end offset = 1
     String fileName = FileNameUtils.fileName(TestUtils.TEST_CONNECTOR_NAME, table, 0, 0, 1);
     // Upload a file on internal stage
     conn.put(stage, fileName, file);
     // Register/Configure a snowpipe client
-    Long clientSequencer = ingestService.configureClient();
-    assert clientSequencer.equals(0l);
+    ConfigureClientResponse configureClientResponse = ingestService.configureClient();
+    assert configureClientResponse.getClientSequencer().equals(0l);
     // Reconfigure the snowpipe client, the clientSequencer should increase
-    clientSequencer = ingestService.configureClient();
-    assert clientSequencer.equals(1l);
+    configureClientResponse = ingestService.configureClient();
+    assert configureClientResponse.getClientSequencer().equals(1l);
     // Get the client offset token
-    String offsetToken = ingestService.getClientStatus();
-    assert offsetToken == null;
+    ClientStatusResponse clientStatusResponse = ingestService.getClientStatus();
+    assert clientStatusResponse.getOffsetToken() == null;
     // Ingest the file with outdated client info should throw error
     ingestService.ingestFilesWithClientInfo(new ArrayList<>(Arrays.asList(fileName)), 0l);
   }
+
+  @Ignore
+  @Test
+  public void ingestMultipleFilesWithClientInfoTest() {
+    String fileContent = "{\"aa\":123}";
+    // start offset = 0, end offset = 1
+    String fileName1 = FileNameUtils.fileName(TestUtils.TEST_CONNECTOR_NAME, table, 0, 0, 1);
+    // start offset = 2, end offset = 3
+    String fileName2 = FileNameUtils.fileName(TestUtils.TEST_CONNECTOR_NAME, table, 0, 2, 3);
+    // start offset = 1, end offset = 2
+    String fileName3 = FileNameUtils.fileName(TestUtils.TEST_CONNECTOR_NAME, table, 0, 1, 2);
```

Comment:
So this is fine for the test but this is not how it happens in code. 
0-1 -> includes two offsets 0 and 1
2-3 -> includes two offsets 2 and 3
3-4 -> includes two offsets 3 and 4

the test looks good though!

Reply:
I see, I'll update it.